.github/workflows/run-simulators.yml

```diff
@@ -42,8 +79,8 @@ jobs:
                 exit 1
             fi
 
-            # wait for status checks to pass 
-            TIMEOUT=300  # Timeout in seconds
+            # wait for status checks to pass
+            TIMEOUT=600  # Timeout in seconds
             START_TIME=$(date +%s)
             END_TIME=$((START_TIME + TIMEOUT))
             while true; do
```

Comment:
This polling loop is a little more complicated, but maybe we can also simplify it by using `aws ec2 wait`?

Reply:
Replaced the instance status polling loop with aws ec2 wait instance-status-ok. Thanks for the recommendation!